content/events/20-21/plaintextaccounting.md

```diff
@@ -1,12 +1,25 @@
 ---
 title: 'Talk: plaintext accounting'
-description: Felix geeft een talk over hoe cool plaintext accounting is
+description: 'Felix en Francis komen praten over plain text accounting'
 created_at: 17-09-2020
-time: '07-11-2017 20:00'
-end: '07-11-2017 22:00'
-location: 'TODO'
+time: '05-10-2020 18:00'
+end: '05-10-2020 20:00'
+location: 'Auditorium A0, Gebouw S9, Campus Sterre (Krijgslaan 281 , 9000 Gent)'
 locationlink: $s9
-soon: true
 ---
 
-TODO
+Accounting of boekhouden is het bijhouden van de stroming van waardevolle goederen zoals geld of tijd. Voornamelijk gebruikt door bedrijven kan je ook boekhouden voor je eigen leven, en ook daar kan het duidelijkheid brengen, helpen om uitgaven te plannen, budget bij te houden en dergelijke.
+
+Plain text accounting is een verzamelnaam voor een aantal tools die een of meerdere eenvoudige tekstbestanden, de journalen, analyseren. Dit maakt ons informatici natuurlijk gelukkig, want dit zorgt er voor dat we de commandoprompt, versiebeheer en het gebruik van onze favoriete tekstverwerker cadeau krijgen.
+
+In deze talk geven Felix en Francis je een korte introductie tot boekhouden, specifiek dubbel boekhouden (*double entry accounting*), met behulp van [hledger](https://hledger.org/) en [ledger](https://ledger-cli.org/).
+
+### Door de coronaregelingen ben je verplicht je in te schrijven.
+
+Schrijf in via deze link: [https://event.fkgent.be/events/299](https://event.fkgent.be/events/299).
+
+### Kan je niet aanwezig zijn?
+
+We zijn aan het zorgen dat deze talk kan worden gestreamd. Houd voor meer info hierover de Zeus [Mattermost](https://chat.zeus.gent) in het oog.
```

Comment:
We zijn aan het zorgen dat deze talk kan worden gestreamd.
-> We proberen ervoor te zorgen dat deze talk kan worden gestreamd.?
@FKD13